OpenRA.Mods.RA/Traits/Wanders.cs

```diff
@@ -0,0 +1,89 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2014 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.RA.Traits
+{
+	[Desc("Wanders around aimlesly when idle.")]
+	abstract class WandersInfo : ITraitInfo
+	{
+		public readonly int WanderMoveRadius = 10;
+
+		[Desc("Number of ticks to wait until decreasing the effective move radius.")]
+		public readonly int MoveReductionRadiusScale = 5;
```

Comment:
The desc for `MoveReductionRadiusScale` is either misleading or inconsistent.
The other two mention that they are timeframes measured in ticks while the name `MoveReductionRadiusScale` sounds like you are adjusting a radius.